CONTRIBUTING.md

```diff
@@ -57,3 +57,11 @@ Here’s how to add an icon to&nbsp;Lawnicons:
 
   Here, the part before the '/' character i.e _org.telegram.messenger_ is the package name [PACKAGE_NAME] and the part after that i.e _org.telegram.messenger_.DefaultIcon is the Activity name [APP_ACIVITY_NAME].
 
+### Using 3rd Party Apps.
+  1. Download [Numix Icon Request] (https://f-droid.org/is/packages/org.numixproject.iconsubmit/). 
```

Comment:
i would use this app for icon requests
https://github.com/Kaiserdragon2/IconRequest/releases

Reply:
@daywalk3r666 yes it's better let me update it,